src/main/java/Dog.java

```diff
@@ -0,0 +1,20 @@
+public class Dog implements Tradable, Domesticatable{
+
+    /**
+     *
```

Comment:
How about a stronger description on what the returned value means? For example "the sound that a dog makes"